### ISSUE TITLE
Panel Accessibility issue

### ISSUE THREAD

**Issue by jame5cross** [SAP/ui5-webcomponents#9471]
### Feature Request Description

The toggle button on the panel component uses a div tag instead of button. Even though role="button" is being applied, this would still be considered inaccessible by many and would be a potential blocker for using the library.

### Proposed Solution

Change the div to a button and remove role="button"

### Priority

None

### Privacy Policy

- [X] I’m not disclosing any internal or sensitive information.

**Comment by TeodorTaushanov** [comment]
Hi @SAP/ui5-webcomponents-topic-rl, can you check this issue?

**Comment by jame5cross** [comment]
Hey @ivoplashkov, thanks for your response.

The <button> element has built-in semantics for interaction and accessibility, therefore if it can be used then it should be used. Basically it is a button so it should be a button, especially in an enterprise level library such as this I'd expect it to be semantically and technically correct.

**Comment by ivoplashkov** [comment]
Hello @jame5cross 

Thank you for the quick reply.

We are obviously aware that the button element has built-in semantics related to the accessibility and interaction, however there is a reason why we have not implemented it. We use a div with the role button to maintain a consistent structure and rendering whether we have a header slot or not. If we were to use a native button, we would need to remove all its styles and create new ones to make it look like a header with text. This would complicate the code and lead to an inconsistent structure in cases with and without the header slot.As described in the W3C documentation, native semantics are preferred, but using ARIA roles is also acceptable, especially when there are visual constraints. More information on the topic could be found here - https://www.w3.org/TR/using-aria/#rule1.

We believe there is no issue with accessibility, as we have met all the requirements specified by ARIA standards for this role. We have included necessary attributes such as role, aria-expanded, and aria-controls, and our tests with various assistive tools and screen readers yield normal results. 

If you still believe there is an issue with accessibility, please provide **_specific information_** on the user impact and a reference to the accessibility standard that is being violated.

Thank you and regards,
Ivaylo
